src/App.js

```diff
@@ -12,12 +12,8 @@ function App() {
         <Navbar />
       </div>
 
-      {/* below are Progress Bar and Direction Buttons placeholders*/}
-      <div className="progress-bar">progress bar</div>
-      <div className="direction-buttons">direction buttons</div>
-
       <div className="posts">
-        <PostsList content={labContent} />
+        <PostsList content={labContent} visibleText={0} />
```

Comment:
Will this always be a constant `0`? If so, no reason to pass it down as a prop

Reply:
It starts at zero, but we're going to set it to different numbers to render more text later using the `setVisibleText` state prop